docker-compose.yaml

```diff
@@ -31,32 +28,29 @@ services:
     restart: on-failure
     volumes:
       - ${STACKS_FOLLOWER_CONFIG_DIRECTORY}:/src/stacks-node
+      - ${STACKS_FOLLOWER_WORKING_DIR_LOCAL}:${STACKS_FOLLOWER_WORKING_DIR}
     ports:
       - ${STACKS_FOLLOWER_RPC_PORT_LOCAL}:${STACKS_FOLLOWER_RPC_PORT}
       - ${STACKS_FOLLOWER_P2P_PORT_LOCAL}:${STACKS_FOLLOWER_P2P_PORT}
-    expose:
-      - ${STACKS_FOLLOWER_RPC_PORT}
-      - ${STACKS_FOLLOWER_P2P_PORT}
     networks:
-      - mainnet
+      - ${COMPOSE_PROJECT_NAME}
     depends_on:
       - api
     command: /bin/stacks-node start --config /src/stacks-node/Config.toml
   api:
     image: ${API_IMAGE}
     container_name: ${API_NAME}
-    restart: on-failure
+    restart: always
+    volumes:
+      - ${API_BNS_DATA_LOCAL}:${API_BNS_DATA}
+      - ./setup-bns.sh:/setup-bns.sh
```

Comment:
Mounting a file does not work consistently across hosts. It is best to always mount folders.

Reply:
hmm, this would require a little refactoring - the idea was to keep the postgres and stacks-blockchain data off of this container. 
I'll come back to this, shouldn't take too long

Reply:
I think the quickest solution would be to create a scripts folder with `setup-ins.sh` in it and then mount that.

Reply:
ha, precisely what i was planning on doing. latest commit does just that - moves the scripts to `./scripts` and mounts that where appropriate.

Reply:
https://github.com/blockstack/stacks-local-dev/commit/71675753e2e89b1014b4287fd3c01a680b6e3fe0